src/Controls/src/Core/HandlerImpl/Window.Impl.cs

```diff
@@ -133,17 +133,20 @@ static void OnPageChanged(BindableObject bindable, object oldValue, object newVa
 
 			if (newPage != null)
 			{
-				newPage.AttachedHandler += OnPageAttachedHandler;
-				newPage.DetachedHandler += OnPageDetachedHandler;
+				newPage.HandlerChanged += OnPageHandlerChanged;
+				newPage.HandlerChanging += OnPageHandlerChanging;
```

Comment:
We don't actually need both anymore. We got the new and the old in Changing so we can unsubscribe from the old and subscribe to the new in a single step.

Reply:
The main reason I like these still inside changed is that they fire after the handler has been set. 

I don't want to subscribe to the AlertManager inside changing, it's ideal if that all wires up after the handler has officially changed